api/src/test/java/org/openmrs/EncounterTest.java

```diff
@@ -1262,4 +1264,25 @@ public void multipleAddingAndRemovingOfSameProvider_shouldNotFail() {
 
 	}
 
+	/**
+	 * @see Encounter#hasDiagnosis(Diagnosis)
+	 */
+	@Test
+	public void hasDiagnosis_shouldCheckIfEncounterHasDiagnosis(){
+		Encounter encounter = new Encounter();
```

Comment:
Can you add another test where a non existing diagnosis should not be found?

Reply:
Okay. I added another assertion of a diagnosis that is not in this encounter. Hence, done

Reply:
I mean a new test method method

Reply:
Okay